__tests__/utils-test.js

```diff
@@ -24,3 +25,51 @@ describe('Unit | utils | getNodeProperty', () => {
     expect(getNodeProperty({}, 'volume.level')).toBe(undefined);
   });
 });
+
+describe('Unit | utils | getNormalizedFilePath', () => {
+  const embroiderBuildPath =
+    '/private/var/folders/abcdefg1234/T/embroider/098765/tests/acceptance/my-test.js';
+  const embroiderBuildPathTwoEmbroiderTokens =
+    '/private/var/folders/embroider/abcdefg1234/T/embroider/098765/tests/acceptance/my-test.js';
+
+  it('returns stripped file path as expected', () => {
+    expect(
+      getNormalizedFilePath({ root: '', filename: embroiderBuildPath })
+    ).toBe('tests/acceptance/my-test.js');
+    expect(
+      getNormalizedFilePath({
+        root: '',
+        filename: embroiderBuildPathTwoEmbroiderTokens,
+      })
+    ).toBe('tests/acceptance/my-test.js');
+  });
+
+  it('returns unmodified file path when path does not include "embroider" as a segment', () => {
+    expect(
+      getNormalizedFilePath({
+        root: '',
+        filename: 'this/is/not-an-embroider/path',
+      })
+    ).toBe('this/is/not-an-embroider/path');
+  });
+});
+
+describe('Unit | utils | getNormalizedFilePath with Windows path', () => {
```

Comment:
This feels odd to do. Why are we doing this vs. just configuring `windows-latest` [in our ci workflow](https://github.com/babel-plugin-ember-test-metadata/babel-plugin-ember-test-metadata/blob/master/.github/workflows/ci-build.yml#L19)?

Reply:
I think if you built these strings with an array and `path.join`, you'd only need one test since `join` should use the appropriate separator for the current CI platform. do add `windows-latest` to the ci workflow tho so we exercise that code path...

Reply:
I think I may need to setup babel-plugin-tester snap shot tests to somehow dynamically point to one snapshot vs another, based on platform (Win vs *Nix).

Reply:
I updated the tests to use path.join.

Adding `windows-latest` to the ci workflow should be follow on work imo, as I may need to create separate output fixtures for Win for each fixture. https://github.com/babel-utils/babel-plugin-tester#outputfixture